### PR TITLE
Edit SSPTemplateValidatorDown

### PR DESCRIPTION
**Pull request by apinnick**
Signed-off-by: Avital Pinnick <apinnick@redhat.com>

### CODE REVIEW

**Comment by apinnick** [comment]
@sradco Ready for review

File: docs/runbooks/SSPTemplateValidatorDown.md [hunk 1]
```diff
@@ -1,30 +1,42 @@
 # SSPTemplateValidatorDown
+<!--apinnick, Nov 2022-->
 
 ## Meaning
 
-The Template Validator is responsible for validating the Virtual Machines do not violate the templates they are assigned to.
-This alert fires when all the Template Validator's pods are down.
+This alert fires when the Template Validator is down.
```
Comment (sradco) [inline docs/runbooks/SSPTemplateValidatorDown.md:6]: @apinnick I think it should remain 
This alert fires when all the Template Validator's pods are down.

File: docs/runbooks/SSPTemplateValidatorDown.md [hunk 2]
```diff
@@ -1,30 +1,42 @@
...
-The Template Validator is responsible for validating the Virtual Machines do not violate the templates they are assigned to.
-This alert fires when all the Template Validator's pods are down.
+This alert fires when all the Template Validator pods are down.
+
+The Template Validator checks virtual machines (VMs) to ensure that they do not violate their templates.
 
 ## Impact
 
-With all Template Validator's pods down the Vms will not be validated against the templates they are associated with.
+VMs are not validated against their templates.
```
Comment (sradco) [inline docs/runbooks/SSPTemplateValidatorDown.md:12]: @borod108 I think we are missing the impact of what will happen if VMs are not validated against their templates.

Comment (borod108) [reply docs/runbooks/SSPTemplateValidatorDown.md:12]: Good point. We can add something like:
This means VMs may be created with specifications that do not much their respective workloads.

**Pull request merged**